packages/contracts/contracts/optimistic-ethereum/OVM/chain/OVM_StateCommitmentChain.sol

```diff
@@ -53,6 +53,7 @@ contract OVM_StateCommitmentChain is iOVM_StateCommitmentChain, Lib_AddressResol
     {
         FRAUD_PROOF_WINDOW = _fraudProofWindow;
         SEQUENCER_PUBLISH_WINDOW = _sequencerPublishWindow;
+        _lastSequencerTimestamp = block.timestamp;
```

Comment:
I think we are going to want to set the blocknumber as well with `block.number`

Reply:
Both exist in the global metadata of the storage container

Reply:
Hmm not sure what you mean. Global metadata only contains `prevTotalElements` and `lastSequencerTimestamp`.